src/control/SConscript

```diff
@@ -129,14 +129,19 @@ def configure_go(denv):
     def check_go_version(context):
         """Check GO Version"""
         context.Display('Checking for Go compiler in $PATH... ')
+
+        global GO_BIN
+
+        GO_BIN = denv.get("GO_BIN", denv.WhereIs('go'))
         if GO_BIN:
             context.Display(GO_BIN + '\n')
         else:
             context.Result(0)
             return 0
 
         context.Display('Checking %s version... ' % GO_BIN)
-        out = os.popen('%s version' % GO_BIN).read()
+        rc = subprocess.run([GO_BIN, 'version'], check=True, shell=False, stdout=subprocess.PIPE)
```

Comment:
(pylint-invalid-name) Variable name "rc" doesn't conform to snake_case naming style